test/unit/lib/plugins/aws/deployFunction/index.test.js

```diff
@@ -20,7 +20,7 @@ describe('AwsDeployFunction', () => {
   let awsDeployFunction;
   let cryptoStub;
 
-  beforeEach(() => {
+  beforeEach(async () => {
```

Comment:
This refactor in general looks good, but we should not invest time into refactor of test files, as most of them are scheduled for general refactor (check issues with `test` label as: https://github.com/serverless/serverless/issues/8594 - currently just for some of files they were created, but in a long run all files should be addressed)